pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -197,6 +212,21 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication {
         return keccak256(abi.encodePacked(actionId, account, where));
     }
 
+    /**
+     * @dev Returns true if `account` is has the permission defined by action `actionId` and target `where`.
+     * This function is specific for the strict permission defined by the tuple `(actionId, where)`, `account` may also
+     * hold the global permission for the action `actionId` which would allow them to perform this action on `where`.
+     * For this reason, it's recommended to use `hasPermission` if checking whether `account` is allowed to perform
+     * a given action.
+     */
+    function isPermissionGranted(
```

Comment:
It would break the name consistency, but would "isPermissionGrantedOnTarget" be clearer?